### PR TITLE
Extracted interfaces for StreamClient, StreamFeed, Collections and Ba…

### PR DESCRIPTION
**Pull request by reddy6ue**
…tchOperations. These interfaces can now be used to unit test any application that uses StreamClient. #14

### CODE REVIEW

File: src/stream-net/IStreamClient.cs [hunk 1]
```diff
@@ -0,0 +1,14 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Stream
+{
+	public interface IStreamClient
+	{
+		BatchOperations Batch { get; }
```
Comment (Link512) [inline src/stream-net/IStreamClient.cs]: Shouldn't this be `IBatchOperations` ?

Comment (reddy6ue) [reply src/stream-net/IStreamClient.cs]: Done! Thanks

**Comment by reddy6ue** [comment]
Can we get this merged please? If nobody has objections......

**Comment by tbarbugli** [comment]
@Link512 ^^^^

**Pull request merged**